test/SimpleSlippageRouter.t.sol

```diff
@@ -0,0 +1,437 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.21;
+
+import { ReentrancyERC4626 } from "src/mocks/ReentrancyERC4626.sol";
+import { CellarAdaptor } from "src/modules/adaptors/Sommelier/CellarAdaptor.sol";
+import { ERC20DebtAdaptor } from "src/mocks/ERC20DebtAdaptor.sol";
+import { MockDataFeed } from "src/mocks/MockDataFeed.sol";
+import { SimpleSlippageRouter } from "src/modules/SimpleSlippageRouter.sol";
+
+// Import Everything from Starter file.
+import "test/resources/MainnetStarter.t.sol";
+import { AdaptorHelperFunctions } from "test/resources/AdaptorHelperFunctions.sol";
+
+/**
+ * TODO: initialShares belongs to  this test contract since it created the cellar no? --> this comment is outlined multiple times because I just want to double check. If it does apply and there is a fix then the asserts marked with this comment need to be adjusted.
```

Comment:
@crispymangoes can you give your quick thoughts on this?

Reply:
Yes shares are minted to msg.sender, so the address deploying the cellar, which is the test contract.

Reply:
We discussed this and it was because the initial shares were being sent to the deployer